### PR TITLE
Add anchor ART attend advanced option

### PR DESCRIPTION
**Pull request by rtesra**

### CODE REVIEW

File: inst/default_options.csv [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+AGO,AGO,1,CY2020Q4,CY2021Q3,CY2015Q4,AGO2015DHS,,,FALSE,FALSE,FALSE,FALSE,FALSE,2020,12,2017,2020,2017,2020,national,TRUE,250,1000,1000,TRUE,-4,TRUE,1,TRUE,1,TRUE,1,none,sex_age_group,none,sex_age_coarse,none,sex_age_coarse,none,sex_age_coarse,logistic,1,TRUE,FALSE,TRUE,FALSE,FALSE,TRUE
```
Comment (r-ash) [inline inst/default_options.csv]: Is the change of format (from `true` -> `TRUE` and `""` -> ``) here from reading in R and then writing out again? I think empty cells will be fine but we should double check, the true TRUE change might cause issues, depends what the JSON serialization does

File: inst/traduire/en-translation.json [hunk 2]
```diff
@@ -96,6 +97,7 @@
   "OPTIONS_ADVANCED_NO_OF_SIMULATIONS_LABEL": "Number of simulations",
   "OPTIONS_ADVANCED_SIMULATION_SEED_LABEL": "Simulation seed",
   "OPTIONS_ADVANCED_LOGIT_MEAN_LABEL": "Logit prior mean for neighboring district attraction",
+    "OPTIONS_ADVANCED_ANCHOR_ART_ATTEND_LABEL": "Anchor resident ART attending",
```
Comment (r-ash) [inline inst/traduire/en-translation.json]: This is pedantic but can we fix the indentation?

File: inst/traduire/fr-translation.json [hunk 3]
```diff
@@ -41,6 +41,7 @@
   "OPTIONS_ADVANCED_NO_OF_SIMULATIONS_HELP": "Nombre de simulations pour la génération de fourchettes d'incertitude",
   "OPTIONS_ADVANCED_SIMULATION_SEED_HELP": "Graine génératrice de nombres aléatoires pour l'échantillon d'incertitude",
   "OPTIONS_ADVANCED_LOGIT_HELP": "Compensation logit relative aux chances d'obtenir un TARV dans le district voisin par rapport au district de résidence",
+    "OPTIONS_ADVANCED_ANCHOR_ART_ATTEND_HELP": "Specify ART attending random effect for residents and neighbours",
```
Comment (r-ash) [inline inst/traduire/fr-translation.json]: More indentation

File: inst/traduire/fr-translation.json [hunk 4]
```diff
@@ -89,6 +90,7 @@
   "OPTIONS_ADVANCED_NO_OF_SIMULATIONS_LABEL": "Nombre de simulations",
   "OPTIONS_ADVANCED_SIMULATION_SEED_LABEL": "Graine de simulation",
   "OPTIONS_ADVANCED_LOGIT_MEAN_LABEL": "Moyenne antérieure Logit relative à l'attractivité des districts voisins",
+      "OPTIONS_ADVANCED_ANCHOR_ART_ATTEND_LABEL": "Anchor resident ART attending",
```
Comment (r-ash) [inline inst/traduire/fr-translation.json]: Indentation again

File: inst/traduire/pt-translation.json [hunk 5]
```diff
@@ -41,6 +41,7 @@
   "OPTIONS_ADVANCED_NO_OF_SIMULATIONS_HELP": "Número de simulações para gerar intervalos de incerteza",
   "OPTIONS_ADVANCED_SIMULATION_SEED_HELP": "Semente geradora de números aleatórios para amostra de incerteza",
   "OPTIONS_ADVANCED_LOGIT_HELP": "Compensação Logit para as probabilidades de obter TARV no distrito vizinho em comparação com o distrito de origem",
+    "OPTIONS_ADVANCED_ANCHOR_ART_ATTEND_HELP": "Specify ART attending random effect for residents and neighbours",
```
Comment (r-ash) [inline inst/traduire/pt-translation.json]:
```suggestion
  "OPTIONS_ADVANCED_ANCHOR_ART_ATTEND_HELP": "Specify ART attending random effect for residents and neighbours",
```

File: inst/traduire/pt-translation.json [hunk 6]
```diff
@@ -89,6 +90,7 @@
   "OPTIONS_ADVANCED_NO_OF_SIMULATIONS_LABEL": "Número de simulações",
   "OPTIONS_ADVANCED_SIMULATION_SEED_LABEL": "Semente de simulação",
   "OPTIONS_ADVANCED_LOGIT_MEAN_LABEL": "Média anterior Logit para atração do distrito vizinho",
+    "OPTIONS_ADVANCED_ANCHOR_ART_ATTEND_LABEL": "Anchor resident ART attending",
```
Comment (r-ash) [inline inst/traduire/pt-translation.json]:
```suggestion
  "OPTIONS_ADVANCED_ANCHOR_ART_ATTEND_LABEL": "Anchor resident ART attending",
```

**Review by r-ash — COMMENTED** [review]
Looks good, a couple of small comments on things. I'll get this deployed and then we can confirm it is looking as intended.

Whilst we're here, do we need to update any of the t2 or t3? Do we want to project to 2023?

File: inst/default_options.csv (hunk 1 above)
Comment (rtesra) [reply inst/default_options.csv]: No I edited in excel and saved back as a csv. What would you suggest as the preferred method?

File: inst/traduire/en-translation.json [hunk 7]
```diff
@@ -44,6 +44,7 @@
   "OPTIONS_ADVANCED_NO_OF_SIMULATIONS_HELP": "Number of simulatons for generating uncertainty ranges",
   "OPTIONS_ADVANCED_SIMULATION_SEED_HELP": "Random number generator seed for uncertainty sample",
   "OPTIONS_ADVANCED_LOGIT_HELP": "Logit offset for odds of obtaining ART in neighbouring district compared to home district",
+  "OPTIONS_ADVANCED_ANCHOR_ART_ATTEND_HELP": "Specify ART attending random effect for residents and neighbours",
```
Comment (r-ash) [inline inst/traduire/en-translation.json:47]: This key isn't used

**Review by r-ash — CHANGES_REQUESTED** [review]
This is current deployed on preview. Looks like the change true -> TRUE is going to be a problem. Not sure what the issue is with excel, my guess it is interpreting the column as a truthy type and converting it. Is there a ways to make it not do that and keep the column as strings? Alternatively we can handle this at the point that this file is read, convert the logical columns to character and then to lower case.

**Comment by r-ash** [comment]
I've updated this to now parse columns from metadata into correct type. So should mean we can stick with the formatting excel wants. Couple of things still to address but I think just small configuration changes.

File: inst/traduire/en-translation.json (hunk 7 above)
Comment (rtesra) [reply inst/traduire/en-translation.json:47]: Thanks - have added to controls.

**Review by r-ash — APPROVED** [review]

**Pull request merged**